static/css/stylesheet.css

```diff
@@ -379,11 +363,23 @@ h1, h2, h3, h4, h5, h6 {
 #btn_files {
     background-position: -822px 0
 }
+
+@font-face {
+    font-family: jolly;
+    src: url("../fonts/Jolly.ttf");
+}
+
 #title {
+    font-family: jolly;
```

Comment:
maybe fall back on comic sans while the font loads? not sure if comic sans ms is actually available on non-windows platforms. not blocking though, you can commit this later